packages/otelbin-validation-image/collector.yaml

```
@@ -13,6 +13,7 @@ exporters:
     endpoint: ingress.eu-west-1.aws.dash0.com:4317
     headers:
       Authorization: "Bearer ${DASH0_AUTHORIZATION_TOKEN}"
+      x-dash0-dataset: "otelbinio"
```

For consistency (and because we don't require the prefix :) )
```suggestion
      Dash0-Dataset: "otelbinio"
```